### DISCUSSION TITLE
Notion sync with Synology NAS (Webdav)

### DISCUSSION THREAD

**Discussion by comeonfeelthenoise** [remotely-save/remotely-save#724] [Q&A]
Hello,

I'm encountering an issue while attempting to synchronize Obsidian with my NAS using the Remotely Save Plugin. Here's a breakdown of what's happened:

1. Successfully synced data from Google Keep to my NAS using the Remotely save Plugin on the same Macbook, confirming no connectivity or settings issues.

2. Attempted to sync data exported from Notion using Obsidian on the same Macbook.

3. Encountered the following error message during synchronization: "unexpectedly keptFolder no decisions: Notion/Life Wiki/Reading List". Found nothing related to Notion contents synced in the NAS storage.

4. Verified and rechecked the Remotely Plugin settings and ensured proper NAS connectivity.

Despite successfully syncing with Google Keep, I'm experiencing this error specifically when syncing Notion exported data. Any insights or assistance on resolving this issue would be greatly appreciated.

Thank you.

**Comment by fyears** [comment]
do you have any files inside Notion/Life Wiki/Reading List folder?

can you export the sync plan, and keep all items in that folder if no sensitive information in the plans?

https://github.com/remotely-save/remotely-save/blob/master/docs/how_to_debug/export_sync_plans.md

**Reply by fyears** [reply]
Actually sometimes the bug might happen if you are running the sync on many devices. For example, in the beginning of device A, something is changed in Onedrive because a new file is uploaded from device B. Then the plugin on device A is confused. Usually you can trigger the sync again and everything should goes well again.
